x/concentrated-liquidity/tick.go

```diff
@@ -87,7 +87,7 @@ func (k Keeper) NextInitializedTick(ctx sdk.Context, poolId uint64, tickIndex in
 	}
 
 	var iter db.Iterator
-	if zeroForOne {
+	if !zeroForOne {
```

Comment:
I think we should also have separate methods here or a strategy design pattern instead of checking `zeroForOne` everywhere

Reply:
Reason: https://softwareengineering.stackexchange.com/questions/147977/is-it-wrong-to-use-a-boolean-parameter-to-determine-behavior